### DISCUSSION TITLE
【委托】紧急求助！

### DISCUSSION THREAD

**Discussion by sdsd122** [share121/inter-knot#1196] [其他]
我重生了,正在变卖全部家产充值鸣潮手游,别人都以为我是傻子,但只有我知道,5月23日这天全球诡异降临.鸣潮手游融合现实,普通人会继承游戏内的属性完成蜕变。
上一世的我没有选择下载鸣潮手游,继而在求生的第十天被一只无冠者偷袭身亡,本以为一生就此结束.不曾想重生回到了开服的前一天.看来我这一世注定是要登上巅峰。
wc，充错了，充进原神了，兄弟借我50，我有大用，球球了

**Comment by gaojiaobei** [comment]
什么？竟然是诡异？这下不得不跑了。

**Comment by FFT66** [comment]
我先跑了！

**Comment by w1-w2-w3** [comment]
我是军医我先跑！
